packages/inferno/src/DOM/props.ts

```diff
@@ -140,9 +140,11 @@ export function patchProp(prop, lastValue, nextValue, dom: Element, isSVG: boole
       }
       break;
     default:
-      if (delegatedEvents[prop]) {
+      const docId = getDocumentId(doc);
+
+      if (docId && delegatedEvents[docId] && delegatedEvents[docId][prop]) {
```

Comment:
This code seems to be little bit wrong. `delegatedEvents[prop]` is used here to check if property is semiSynthetic event or not. This will be constant information for all documents and no docId should be needed

Reply:
I reverted this part